kentik_api_library/kentik_api/requests_payload/devices_payload.py

```diff
@@ -466,7 +465,7 @@ def from_json(cls, json_string: str):
         )
 
     def to_applied_labels(self) -> AppliedLabels:
-        labels = [l.to_device_label() for l in self.labels]
+        labels = [label.to_device_label() for label in self.labels]
```

Comment:
Ditto

Reply:
Still to do

Reply:
Do You think label here is not readble?

Reply:
I agree. Whatever linter forces us to use "descriptive" variable names in list comprehensions should be disabled. The longer name is not wrong, but it is also not any better than the short one and the general rule has a good chance to force longer list comprehensions to require line-wrapping.